# Shelf parameters
shelf_distance_from_robot: 1.0

shelf_width: 0.873
shelf_height: 2.37
shelf_depth: 0.875
shelf_wall_width: 0.02

# Bin parameters
first_bin_from_bottom: 0.98
#first_bin_from_bottom: 0.81 # If at correct competition height
first_bin_from_right: 0.025
bin_width: 0.250825
bin_middle_width: 0.2921

bin_short_height: 0.2286
bin_tall_height: 0.2667

bin_depth: 0.42
bin_top_margin: 0.01
bin_left_margin: 0.01
num_bins: 12.0

# Execution Speed
main_velocity_scaling_factor: 0.5 # during planning
approach_velocity_scaling_factor: 0.25
lift_velocity_scaling_factor: 0.5
retreat_velocity_scaling_factor: 0.8

# Sleep timers
wait_before_grasp: 0.1
wait_after_grasp: 0.1

# Goal bin
goal_bin_x: -0.6
goal_bin_y: 1.02
goal_bin_z: 0.98